source/userguide/internationalization.rst

```diff
@@ -0,0 +1,97 @@
+Internationalization support
+============================
+
+Mindmeld supports most languages that can be tokenized like English. If the language does not use spaces between
+words or has many non-English-like punctuation marks, pre-process the data to remove punctuations and add spaces between words.
```

Comment:
@serapio : can we give an example of the languages that have many of these punctuation marks?

Reply:
This is something I had heard was a problem generally with using \W regex across languages, but the ones I tried in German, Arabic and Chinese were all handled correctly.